js/gated-content/src/components/blog/BlogListing.vue

```diff
@@ -108,6 +109,16 @@ export default {
     $route: 'load',
     excludedVideoId: 'load',
     sort: 'load',
+    isCategoriesLoaded() {
+      if (this.categories !== null) {
```

Comment:
In case we have empty array default value:
`if (Array.isArray(this.categories) && this.categories.length)`